packages/ui-components/package.json

```diff
@@ -63,10 +63,10 @@
 	},
 	"devDependencies": {
```

Comment:
@GoncaloCamaz Do you check if our current project dependencies needs some updates?

Reply:
Done and merged into this PR @miguelpinto98